test/runner.c

```diff
@@ -83,6 +83,7 @@ main (int argc, char * argv[])
   jobjectArray argv_value;
   jstring data_dir_value, cache_dir_value;
   guint arg_index;
+  gboolean optimization_enabled = false;
```

Comment:
```suggestion
  gboolean optimization_enabled = FALSE;
```

(As it's a `gboolean` and not a `bool`.)